readthedocs/search/views.py

```diff
@@ -143,7 +155,7 @@ def elastic_search(request, project_slug=None):
                 'user': request.user,
                 'project': user_input.project or '',
                 'type': user_input.type or '',
-                'version': user_input.version or '',
+                'version': user_input.version if hasattr(user_input, "version") else 'default',
```

Comment:
is this still needed?

Reply:
yes, because `version` is not always in `user_input`